xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/Registration.xml

```diff
@@ -295,7 +296,7 @@
         },
         'noReturn' : true
       })
-    #set($discard = $fields.add($field))
+    #set($discard = $aboutYouFields.add($field))
```

Comment:
It doesn't feel accurate to put the captcha in the about you fields, also you didn't provided any screenshot for registration with captcha?

Reply:
The semantics are a bit wrong yup, however I'm not sure how we'd want to show it so I kept it similar to what was done before. It's also more convenient because I don't need to update how we display the captcha (here it's treated the same as any other field).
![image](https://github.com/user-attachments/assets/5a4f5925-ddb6-41da-85c0-d024ec36d320)


@tkrieck Do you think this is an okay position for the captcha? Do you have an idea for a design that would clearly separate it from the rest of the form without looking too cluttered?

____
If we don't do anything right now I can still open a low importance ticket for accessibility because of the wrong semantics. IMO this is not the core of the change and we can wait until later to improve it.

Reply:
@Sereza7 The positioning doesn't bother me, it looks okay. The input can get a new label though, could it be just "Captcha"? The little help beneath the captcha field can receive the current applied label (Please validate the captcha to prove you are not a robot)

Reply:
Thanks for your help!
Addressed in https://github.com/xwiki/xwiki-platform/pull/3155/commits/c49f94d13e0de69b92c44eb099c89279b924b56e :+1: 
Here's what this looks like now vvv
![Screenshot from 2024-10-02 11-22-11](https://github.com/user-attachments/assets/e492136a-86ad-46e6-8672-5b61a9c46062)